### PR TITLE
Test react

### PR DESCRIPTION
**Pull request by Guy-Gustave**
A pull request from dev-branch, where i made tests.

### CODE REVIEW

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @Guy-Gustave :wave: 

### Thank you for making the changes requested by the previous TSE, but you still have some linter errors, please fix them before requesting a new review.

Kindly make the requested changes and submit them for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

File: src/Affiche.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
```
Comment (Torres-ssf) [inline src/Affiche.js:1]: **OPTIONAL**
- [ ] Let's remove the `eslint` disable line here

File: src/__tests__/calculatate.test.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+/* eslint-disable */
```
Comment (Torres-ssf) [inline src/__tests__/calculatate.test.js:1]: **OPTIONAL**
- [ ] Let's remove the `eslint` disable line here

File: src/__tests__/calculatate.test.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+const button1 = '+/-';
+const button2 = '+';
+const button3 = '-';
+const button5 = '/';
+const button6 = 'AC';
+const calculatorObj1 = { total: 15, next: 5, operation: button6 };
+const calculatorObj2 = { total: 15, next: 20, operation: button2 };
+const calculatorObj3 = { total: 15, next: 20, operation: button3 };
+// const calculatorObj4 = { total: 15, next: 20, operation: button4 };
```
Comment (Torres-ssf) [inline src/__tests__/calculatate.test.js:13]: **OPTIONAL**
- [ ] Let's remove the `commented code here

File: src/components/ButtonPanel.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
 /* eslint-disable */
```
Comment (Torres-ssf) [inline src/components/ButtonPanel.js:1]: **OPTIONAL**
- [ ] Let's remove the `eslint` disable line here

File: src/components/NavBar.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import '../index.css';
+/* eslint-disable */
```
Comment (Torres-ssf) [inline src/components/NavBar.js:4]: **OPTIONAL**
- [ ] Let's remove the `eslint` disable line here

File: src/index.js [hunk 6]
```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './components/App';
+// import App from './components/App';
```
Comment (Torres-ssf) [inline src/index.js:4]: **OPTIONAL**
- [ ] Let's remove the commented code here

File: src/pages/Quote.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable */
```
Comment (Torres-ssf) [inline src/pages/Quote.js:1]: **OPTIONAL**
- [ ] Let's remove the `eslint` disable line here

**Review by Torres-ssf — APPROVED** [review]
### PR Approved :+1: :tada: 
Hello @Guy-Gustave :hand: 
The linter errors were fixed, nice job :+1: :clap: 
You may merge the PR :smile: 

**OPTIONAL**
- [ ] Since the last reviewer only requested to fix linter errors, the PR approved. However there still some changes asked in the first review (all related to removing `eslint` disable comment line) [here](https://github.com/Guy-Gustave/calculator/pull/9) that we need to change.
Screenshot from 2021-04-12 11-10-56


Happy coding :computer: 
**Sérgio Torres**

**Pull request merged**